src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Status.tsx

```diff
@@ -71,38 +69,26 @@ export const Status: React.FC<StatusProps> = ({ accountId }) => {
     selectMarketDataByAssetIdUserCurrency(state, assetId ?? ''),
   )
 
-  const account = useMemo(() => accountId && fromAccountId(accountId).account, [accountId])
-
-  const serializedTxIndex = useMemo(() => {
-    if (!(state?.txid && accountId && account)) return ''
-    return serializeTxIndex(accountId, state.txid, account)
-  }, [state?.txid, accountId, account])
-
-  const confirmedTransaction = useAppSelector(gs => selectTxById(gs, serializedTxIndex))
-
   useEffect(() => {
-    if (!accountId) return
+    if (!(contextDispatch && state?.txid?.length)) return
```

Comment:
nitpick but keep consistent between deposit and withdraw with either `state?.txid` or `state?.txId?.length`

Reply:
Done in [1ff5536](https://github.com/shapeshift/web/pull/7722/commits/1ff5536785768da4146e3b3527564b791b87790f)